modules/yardstick/pom.xml

```diff
@@ -58,12 +58,6 @@
             <version>${project.version}</version>
         </dependency>
 
-        <dependency>
-            <groupId>org.gridgain</groupId>
-            <artifactId>ignite-log4j</artifactId>
-            <version>${project.version}</version>
```

Comment:
Let's replace it with log4j2 instead. I'm afraid we'll lose decent logging in benchmarks if we just remove this module.